ncbi-submit/.gitignore

```diff
@@ -1,3 +1,9 @@
 *.pyc
 testing/
 Dockerfile.local
+bpt/
+bpt.json
+sbt.json
+sqn/
+sqn.json
+sqn.tar.gz
```

Comment:
I'm not sure if these changes were intended to be included, since they're in a different directory.

Reply:
I did intend to include these changes, admittedly primarily out of laziness. The files are on my computer and I wanted to update `.gitignore` in this directory so that I could worry a little bit less about actually committing them. 😆

Reply:
lol, that works